fontbe/src/metrics_and_limits.rs

```diff
@@ -184,3 +188,27 @@ impl Work<Context, Error> for MetricAndLimitWork {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use kurbo::BezPath;
+    use write_fonts::tables::glyf::SimpleGlyph;
+
+    use super::GlyphLimits;
+
+    // advance 0, bbox (-437,611) => (-334, 715) encountered in NotoSansKayahLi.designspace
+    #[test]
+    fn negative_xmax_does_not_crash() {
+        let mut glyph_limits = GlyphLimits::default();
+        // path crafted to give the desired bbox
+        glyph_limits.update(
```

Comment:
should you not assert the result is as expected instead of simply not crashing?

Reply:
Fixed